### ISSUE TITLE
Show log returns empty window with error: No such option: --graph 

### ISSUE THREAD

**Issue by andreatitolo** [koordinates/kart-qgis-plugin#92]
**Describe the bug**
Attempting to show the log for any repositories I have first returns a "Kart failed" error, then displays an empty log, no graph.

```
Kart failed with the following message: 
Usage: kart_cli log [OPTIONS] [REVISIONS] [--] [FILTERS]
Try 'kart_cli log --help' for help.
Error: No such option: --graph 
```

However, in the terminal, `kart log` shows the correct log.

**To Reproduce**
1. Open QGIS with the Kart repositories pane
2. Click on any repo and "Show Log.."
3. Return the above error 
4. Kart plugin displays an empty log

**Expected behavior**
I expected to see the log graph.

**Version info (this is important!):**
 - Operating System (e.g. Windows 10, macOS): MacOS Ventura 13.2
 - QGIS, Kart & Plugin details: QGIS 3.28.2, Kart version 0.12.0, Kart plugin version 1.0.4
  ```
Kart Plugin version
1.0.4
QGIS version
3.28.2-Firenze
Operating system
macOS 13.2 x86_64 i386 64bit
Kart version
Kart v0.12.0, Copyright (c) Kart Contributors
» GDAL v3.6.2; PROJ v9.1.1; PDAL v2.4.3
» PyGit2 v1.9.0; Libgit2 v1.4.2; Git v2.38.1; Git LFS v3.3.0
» SQLAlchemy v1.4.45; pysqlite3 v2.6.0/v3.40.0; SpatiaLite v5.0.1; Libpq v14.0.4
```

**Additional context**
I haven't used kart for a two-three weeks, but I remember having no troubles seeing the log before. I updated to 0.12.0 and this issue started to show up (I guess?).

**Comment by olsen232** [comment]
@craigds selected Kart 0.12 as the time when unknown options would no longer be supported by Kart log, and did some prep for it but we missed this particular invocation. He's away today but I won't work against his plan - instead, to move forward, the following is required:
1. Add --graph to the options kart log supports - see https://github.com/koordinates/kart/pull/784, already merged
2. Use `--output-format=text:FORMAT` or `-otext:FORMAT` instead of `--format=FORMAT` when calling kart log from the plugin - see https://github.com/koordinates/kart-qgis-plugin/pull/93
3. Release new Kart and new plugin

**Comment by olsen232** [comment]
Fixed as of v1.0.6, which works with Kart 0.12.2
